### PR TITLE
Added move data JSON

### PR DESCRIPTION
**Pull request by AnonymousRandomPerson**

### CODE REVIEW

File: include/constants/move_id.h [hunk 1]
```diff
@@ -402,11 +406,14 @@
...
 #define MOVE_POSSESS 0x192
 #define MOVE_ITEMIZE 0x193
+#define MOVE_MD1 0x194 // Unused
+#define MOVE_PROJECTILE 0x195
 #define MOVE_HURL 0x196
 #define MOVE_MOBILE 0x197
 #define MOVE_ITEM_TOSS 0x198
 #define MOVE_SEE_STAIRS 0x199
 #define MOVE_LONG_TOSS 0x19A
+#define MOVE_MD1_2 0x19B // Unused
```
Comment (SethBarberee) [inline include/constants/move_id.h:416]: Whats the `MD1` Prefix mean here?

File: include/constants/move_id.h [hunk 2]
```diff
@@ -357,6 +357,10 @@
 #define MOVE_AEROBLAST 0x161
 #define MOVE_VOLT_TACKLE 0x162
 #define MOVE_REGULAR_ATTACK 0x163
+#define MOVE_IS_WATCHING 0x164
+#define MOVE_BIDE_2 0x165 // Unused
+#define MOVE_DOLLAR_1 0x166 // Unused
```
Comment (SethBarberee) [inline include/constants/move_id.h:362]: why `DOLLAR`?

**Review by SethBarberee — COMMENTED** [review]
Looks fine. Mainly just asking about defines for understanding of the names

File: include/constants/move_id.h (hunk 1 above)
Comment (PikalaxALT) [reply include/constants/move_id.h:416]: https://github.com/pret/pmd-red/pull/60/files#diff-059e3ff7eb81a0797b026f1fd75cf0bb9cdf5c956b1b0e0975a75b7ffd0fe326R8293

File: include/constants/move_id.h (hunk 2 above)
Comment (PikalaxALT) [reply include/constants/move_id.h:362]: https://github.com/pret/pmd-red/pull/60/files#diff-059e3ff7eb81a0797b026f1fd75cf0bb9cdf5c956b1b0e0975a75b7ffd0fe326R7313

File: include/constants/move_id.h (hunk 1 above)
Comment (SethBarberee) [reply include/constants/move_id.h:416]: :+1: I see

File: include/constants/move_id.h (hunk 2 above)
Comment (SethBarberee) [reply include/constants/move_id.h:362]: :+1: Chunsoft being chunsoft

**Comment by SethBarberee** [comment]
Thanks!

**Pull request merged**